### PR TITLE
Blog post about MicroProfile GraphQL

### PR DESCRIPTION
**Pull request by phillip-kruger**
Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

**Comment by phillip-kruger** [comment]
@emmanuelbernard Any feedback?

File: _data/authors.yaml [hunk 1]
```diff
@@ -144,7 +144,7 @@ cesarsaavedra:
...
+  bio: "Cesar Saavedra focuses on Eclipse MicroProfile, OpenJDK, Jakarta EE, and Quarkus. He has been a technical seller, consultant, and developer and now enjoys doing technical product marketing for Red Hat Middleware. In addition, he is a MicroProfile committer, has authored a book, white papers, blog posts, and participated in conferences as a speaker and delivered webinars to end customers, partners and internal audiences. He has over 28 years of IT experience, many of them in the application server market, hybrid integration, API management and Process Automation. Before Red Hat, Cesar was at IBM, Oracle, BEA Systems, TIBCO, Lucent Technologies, and AT&T Bell Laboratories. In his spare time, Cesar enjoys spending time with his family, traveling and playing soccer."
```
Comment (emmanuelbernard) [inline _data/authors.yaml:147]: Weird there is a diff here. do you know why @phillip-kruger

Comment (phillip-kruger) [reply _data/authors.yaml:147]: ??? W.t.f ? Hold on I am looking.

File: _posts/2020-06-11-supersonic-subatomic-graphql.adoc [hunk 2]
```diff
@@ -0,0 +1,134 @@
+---
```
[the remark this answers is not in the document]
Comment (phillip-kruger) [reply _posts/2020-06-11-supersonic-subatomic-graphql.adoc:1]: Something weird, apologies. Let me check

[the remark this answers is not in the document]
Comment (phillip-kruger) [reply _posts/2020-06-11-supersonic-subatomic-graphql.adoc:1]: Right, I was using Atom IDE (that I dont's use often) and never saved the file it seem. (And never checked here) Sorry.

**Comment by emmanuelbernard** [comment]
I adapted a bit the formatting and pushed. It is a great post @phillip-kruger !

**Comment by emmanuelbernard** [comment]
https://twitter.com/QuarkusIO/status/1275457076014301184

**Comment by phillip-kruger** [comment]
Thanks @emmanuelbernard !

**Pull request closed without merging**